tasks/linter.py

```diff
@@ -383,7 +383,7 @@ def __repr__(self):
 def list_get_parameter_calls(file):
     aws_ssm_call = re.compile(r"^.+ssm get-parameter.+--name +(?P<param>[^ ]+).*$")
     # remove the first letter of the script name because '\f' is badly interpreted for windows paths
-    wrapper_call = re.compile(r"^.+etch_secret.(sh|ps1)[\"]? +(?P<param>[^ )]+).*$")
+    wrapper_call = re.compile(r"^.+etch_secret.(sh|ps1)[\"]? (-parameterName )?+(?P<param>[^ )]+).*$")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Should this match `vault` parameters? As it creates `SSMParameterCall`s maybe the `vault` parameters must be ignored. Otherwise, do we have to update this function and split `ssm` / `vault` parameters?